fms2_io/fms_io_utils.F90

```diff
@@ -510,8 +510,9 @@ subroutine ascii_read(ascii_filename, ascii_var, out_dims)
   character(len=:), dimension(:), allocatable, intent(out) :: ascii_var !< The
                                                                         !! string
                                                                         !! array
-  integer, dimension(2), optional, intent(out) :: out_dims
+  integer, dimension(2), optional, intent(out) :: out_dims !< Optional argument to return number of lines and max_length of file
```

Comment:
This is *ok*, but why not just have 2 different optional arguments (`number_of_lines` and `max_length_of_file`) instead of one array with two different pieces of information?